tools/chainbuilder/main.go

```diff
@@ -87,10 +98,14 @@ func main() {
 
 	rootCmd.Flags().Int("num-blocks", 100, "Number of blocks to generate")
 	rootCmd.Flags().Int("block-size", appconsts.DefaultMaxBytes, "Size of each block in bytes")
-	rootCmd.Flags().Int("square-size", appconsts.DefaultSquareSizeUpperBound, "Size of the square")
 	rootCmd.Flags().Duration("block-interval", time.Second, "Interval between blocks")
 	rootCmd.Flags().String("existing-dir", "", "Existing directory to load chain from")
 	rootCmd.Flags().String("namespace", "", "Custom namespace for the chain")
+	rootCmd.Flags().Bool("up-to-now", false, "Tool will terminate if the block time reaches the current time")
+	rootCmd.Flags().Uint64("app-version", appconsts.LatestVersion, "App version to use for the chain")
+	rootCmd.Flags().String("chain-id", "", "Chain ID to use for the chain. Defaults to a random 6 character string")
+	rootCmd.SilenceUsage = true
+	rootCmd.SilenceErrors = true
```

Comment:
[question] out of curiosity, why silence the usage message and errors? They seem helpful if a user provides incorrect inputs to the tool.

Reply:
They can still use the --help flag if they want to understand how to use it. I personally just find it unhelpful/confusing if I have an error and it prints out these lines on generic usage that has nothing to do with the error